README.md

```diff
@@ -4,17 +4,49 @@
 
 A simple Julia wrapper around the [PROJ](https://proj.org/) cartographic projections library.
 
-Basic example:
+Quickstart, based on the [PROJ docs](https://proj.org/development/quickstart.html):
 
 ```julia
 using Proj4
 
+# Proj4.jl implements the CoordinateTransformations.jl API
+# A Proj4.Transformation needs the source and target coordinate reference systems
+trans = Proj4.Transformation("EPSG:4326", "+proj=utm +zone=32 +datum=WGS84")
+
+# Once created, you can call this object to transform points
+# the result will be a SVector from StaticArrays.jl
+# Here the (latitude, longitude) of Copenhagen is entered
+trans([55, 12])
+# -> SVector{2, Float64}(691875.632, 6098907.825)
+
+# Note that above the latitude is passed first, because that is the axis order that the
+# EPSG mandates. If you want to pass in (longitude, latitude) / (x, y), you can set the
+# `always_xy` keyword to true. For more info see https://proj.org/faq.html#why-is-the-axis-ordering-in-proj-not-consistent
+trans = Proj4.Transformation("EPSG:4326", "+proj=utm +zone=32 +datum=WGS84", always_xy=true)
+
+# now we input (longitude, latitude), and get the same result as above
+trans([12, 55])
+# -> SVector{2, Float64}(691875.632, 6098907.825)
+
+# using `inv` we can reverse the direction, `compose` can combine two transformations in one
+inv(trans)([691875.632, 6098907.825]) ≈ [12, 55]
+
+
```

Comment:
redundant whitespace?

Reply:
Yeah wanted to keep them apart, separate code blocks are better: c7b0c522bee9dd46abbb8d567687270d7c3e1e52